[inductor] Fix an issue causing "Could not generate fp64 outputs"

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #92036

Summary: Fix a fp64 version of model failed-to-run issue when convert_element_type
appears in the model. The failure can cause some numerical difference
recognized as accuracy error since the fp64 baseline result is not
available, and thus distracts Minifier from finding a real culprit for
accuracy error.

See the discussion in https://github.com/pytorch/pytorch/issues/93595

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx

desertfire committed:
[inductor] Fix an issue causing "Could not generate fp64 outputs"

Summary: Fix a fp64 version of model failed-to-run issue when convert_element_type
appears in the model. The failure can cause some numerical difference
recognized as accuracy error since the fp64 baseline result is not
available, and thus distracts Minifier from finding a real culprit for
accuracy error.

See the discussion in https://github.com/pytorch/torchdynamo/issues/1812

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge -r

desertfire committed:
Update on "[inductor] Fix an issue causing "Could not generate fp64 outputs""


Summary: Fix a fp64 version of model failed-to-run issue when convert_element_type
appears in the model. The failure can cause some numerical difference
recognized as accuracy error since the fp64 baseline result is not
available, and thus distracts Minifier from finding a real culprit for
accuracy error.

See the discussion in https://github.com/pytorch/torchdynamo/issues/1812

cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx

[ghstack-poisoned]

desertfire commented:
/easycla

desertfire commented:
@pytorchbot merge -f "CLA test broken"

ngimel commented:
You can't force merge without CLA. Did you check that email used in this commits matches the email you signed CLA with?

desertfire committed:
Update on "[inductor] Fix an issue causing "Could not generate fp64 outputs""


Summary: Fix a fp64 version of model failed-to-run issue when convert_element_type
appears in the model. The failure can cause some numerical difference
recognized as accuracy error since the fp64 baseline result is not
available, and thus distracts Minifier from finding a real culprit for
accuracy error.

See the discussion in https://github.com/pytorch/torchdynamo/issues/1812

cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge -f "rocm failure is irrelevant"
